tutorials/workzone-build-5-url-app/workzone-build-5-url-app.md

```diff
@@ -8,18 +8,18 @@ author_name: Lindsay Bert
 author_profile: https://github.com/LindsayBert
 ---
 
-# Add a URL App to Your SAP Work Zone Page
-<!-- description --> Create a URL app and add it to a page in SAP Work Zone.
+# Add a URL App to Your Workpage in SAP Build Work Zone, advanced edition
```

Comment:
Please apply title case to "advanced edition"

Reply:
Hi Michael - this is the brand name from Brand Voice and we've been told not to capitalize advanced addition